changelog/aliasdecly_func.dd

```diff
@@ -0,0 +1,20 @@
+AliasDeclarationY allows to alias function types
```

Comment:
1. What is `AliasDeclarationY` ?
2. Need to precisely express grammar changes using BNF, like the spec does, not handwaving.

Reply:
Also the statement makes no grammatical sense.